backend/external/imagebb.js

```diff
@@ -1,8 +1,22 @@
 const fetch = require('node-fetch');
 const FormData = require('form-data');
 
-const API_KEY = process.env.IMAGE_BB_API_KEY;
-const api = `https://api.imgbb.com/1/upload`;
+const IMGUR = 'imgur'
+const IMGBB = 'imgbb'
```

Comment:
I would suggest a separate file for _imgur_ handling as this file name is `imagebb.js`